src/components/AddBookForm.js

```diff
@@ -0,0 +1,16 @@
+export default function AddBookForm() {
+  return (
+    <form>
+      <h3>ADD NEW BOOK</h3>
+      <input
+        type="text"
+        placeholder="Book title"
+      />
+
+      <input
+        type="text"
+        placeholder="Author"
+      />
+    </form>
+  );
```

Comment:
- [ ] Please add a `button` to submit as a placeholder for now

Reply:
@DeliceLydia 
Let me add that right away.
Thanks for the feedback